### ISSUE TITLE
About setting 'num_epoch'

### ISSUE THREAD

**Issue by dzhyy** [xuebinqin/U-2-Net#237]
Hello, I want to use the pre training model for fine-tune to adapt to my portrait matting task. I found the original  value of epoch is set to a very large value (100000). If it is fine-tune on the original basis, how much  'num_epoch' set  is appropriate? thank you for answer.

**Comment by baleksey** [comment]
@dzhyy I've trained my last model for portraits for about 130-150 epoches until it just stops decreasing loss. 100k is definitely too much. Meaningful results are already after 50-70 epoches.

The final training loss was ~ 0.146 and portrait segmentation is pretty good. So you can use those numbers as reference for your training.
